src/interfaces/routes.ts

```diff
@@ -28,8 +28,13 @@ export interface GetTransferHistoryRequest {
   action?: string; //Deposit / Withdraw
 }
 export interface GetOrderRequest extends GetTransactionHistoryRequest {
+  symbol?: MarketSymbol;
+  orderId?: number;
+  orderHashes?: string[];
   statuses: ORDER_STATUS[]; // status of orders to be fetched
```

Comment:
Is there a way that we can query for a list of hashes and then see the different `statuses` of each order? Would this require DB schema changes or do you think it can all be done just here in the client code?

Reply:
That'd be a different API call as in get orders route we only return orders with it's current status, If the user wants to see trail of `statuses` of each order we can give them as we store it in different `OrderStatuses` table

Reply:
Ok I see, that makes sense. If MM team needs that we can do it in a separate PR